mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageIdDAOTest.java

```diff
@@ -41,8 +43,12 @@
 import reactor.core.publisher.Flux;
 
 class CassandraMessageIdDAOTest {
+    public static final CassandraModule MODULE = CassandraModule.aggregateModules(
+        CassandraMessageModule.MODULE,
+        CassandraSchemaVersionModule.MODULE);
+
     @RegisterExtension
-    static CassandraClusterExtension cassandraCluster = new CassandraClusterExtension(CassandraMessageModule.MODULE);
+    static CassandraClusterExtension cassandraCluster = new CassandraClusterExtension(MODULE);
```

Comment:
unrelated to this commit?

Reply:
Necessary to run only this test class

Reply:
ok, but next time you could have separated this into a proper commit

Reply:
At one point we either have to either do it everywhere, on add a neat helper.